tests/conftest.py

```diff
@@ -161,3 +162,11 @@ async def echo_sse_server() -> AsyncGenerator[dict[str, str]]:
         # Clean up the process when test is done
         process.kill()
         await process.wait()
+
+
+@pytest.fixture
+def mock_litellm_response() -> ModelResponse:
+    """Fixture to create a standard mock LiteLLM response"""
+    return ModelResponse.model_validate_json(
+        '{"id":"chatcmpl-BWnfbHWPsQp05roQ06LAD1mZ9tOjT","created":1747157127,"model":"gpt-4o-2024-08-06","object":"chat.completion","system_fingerprint":"fp_f5bdcc3276","choices":[{"finish_reason":"stop","index":0,"message":{"content":"The state capital of Pennsylvania is Harrisburg.","role":"assistant","tool_calls":null,"function_call":null,"annotations":[]}}],"usage":{"completion_tokens":11,"prompt_tokens":138,"total_tokens":149,"completion_tokens_details":{"accepted_prediction_tokens":0,"audio_tokens":0,"reasoning_tokens":0,"rejected_prediction_tokens":0},"prompt_tokens_details":{"audio_tokens":0,"cached_tokens":0}},"service_tier":"default"}'
```

Comment:
In Lumigator we used files to store this kind of tests instead of embedding them in the code, wdyt?

Reply:
In any-agent we've been putting them directly into conftest etc. Hmm for the moment I'll keep it as is but I'll think about changing this in the future. Thanks!